src/libexpr/primops/fetchClosure.cc

```diff
@@ -137,7 +194,7 @@ static RegisterPrimOp primop_fetchClosure({
       `/nix/store/ldbh...`.
 
       If `fromPath` is already content-addressed, or if you are
-      allowing impure evaluation (`--impure`), then `toPath` may be
+      allowing input addressing (`inputAddressed = true;`), then `toPath` may be
```

Comment:
"allowing input addressing" is a bit vague, maybe this should say something like "if you allow paths built by input-addressed derivation (i.e. derivations that don't have `__contentAddressed = true`)".

Reply:
I think we can overhaul the docs more while still promoting content-addressing?